Protobuild.toml

```diff
@@ -19,7 +19,7 @@ plugins = ["grpc"]
   # This is the default.
   after = ["/usr/local/include"]
 
-# This section let's us map protobuf imports to Go packages. These will become
+# This section let's map protobuf imports to Go packages. These will become
```

Comment:
Should be `lets us`.

We conjugate `let` to `lets`. I accidentally used possessive here because of muscle memory.

Should probably be more active: 'This section maps protobuf imports to Go packages`.